README.md

```diff
@@ -23,12 +23,10 @@
 |                     |                       |
 | 11                  | 2946529113426         |
 |                     |                       |
-| 14                  | 8472751160746         |
-|                     |                       |
-| 15                  |                       |
-|                     |                       |
-| 15                  | 9637446223215                      |
```

Comment:
@wojtekerbetowski Is this change related to the PR?

Reply:
I had to clean up a bit!